specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -2800,6 +2834,10 @@
             "modelAsString": true
           }
         },
+        "purchasePlan": {
+          "$ref": "#/definitions/PurchasePlan",
+          "description": "Purchase Plan Information of the Virtual Machine persisted on the Disk."
```

Comment:
Purchase Plan Information of the image used to create the Disk?

Reply:
> Purchase Plan Information of the image used to create the Disk?

Optional field, if the Customer wants to put specific Purchase Plan then they can.

Reply:
might need a better description, please ask Vikram about it. Thanks

Reply:
Purchase plan information for the the image from which the OS disk was created.

Reply:
add examples